api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -91,7 +91,10 @@ message ExtAuthz {
   type.v3.HttpStatus status_on_error = 7;
 
   // Specifies a list of metadata namespaces whose values, if present, will be passed to the
-  // ext_authz service. :ref:`filter_metadata <envoy_v3_api_field_config.core.v3.Metadata.filter_metadata>` is passed as an opaque ``protobuf::Struct``.
+  // ext_authz service. The :ref:`filter_metadata <envoy_v3_api_field_config.core.v3.Metadata.filter_metadata>`
+  // is passed as an opaque ``protobuf::Struct``.
+  //
+  // Please note that this field exclusively applies to the gRPC ExtAuthZ service and has no effect on the HTTP service.
```

Comment:
```suggestion
  // Please note that this field exclusively applies to the gRPC ext_authz service and has no effect on the HTTP service.
```

not sure what the proper name for this should be but its ext_authz everywhere else affaict